R/pkg/R/DataFrame.R

```diff
@@ -984,12 +984,12 @@ setMethod("unique",
 #' of the total count of of the given SparkDataFrame.
 #'
 #' @param x A SparkDataFrame
-#' @param withReplacement Sampling with replacement or not
+#' @param withReplacement Sampling with replacement or not. Default is \code{FALSE}.
```

Comment:
we actually want to change to not documenting the default value if it is already in the signature - because then it would be the roxygen2 generated doc